src/components/scenes/newAccount/NewAccountUsernameScene.tsx

```diff
@@ -43,6 +41,13 @@ export const NewAccountUsernameScene = ({ branding }: Props) => {
     false
   )
 
+  const initUsername = useSelector(state => state.create.username ?? '')
```

Comment:
Go ahead and remove the `?? ''` bit, and do it instead inside the `useState` hook.